powerauth-push-client/src/main/java/io/getlime/push/client/PushServerClient.java

```diff
@@ -145,16 +151,39 @@ public boolean createDevice(String appId, String token, MobilePlatform platform,
         return response.getStatus().equals(Response.Status.OK);
     }
 
+    /**
+     * Register a device to the push server.
+     *
+     * @param request Create device request.
+     * @return True if device registration was successful, false otherwise.
+     * @throws PushServerClientException In case of network, response / JSON processing, or other IO error.
+     */
+    public boolean createDevice(final CreateDeviceRequest request) throws PushServerClientException {
+        String error = CreateDeviceRequestValidator.validate(request);
+        if (error != null) {
+            throw new PushServerClientException(error);
+        }
+
+        logger.info("Calling create device service, appId: {}, token: {}, platform: {} - start", request.getAppId(), maskToken(request.getToken()), request.getPlatform());
```

Comment:
Well, apart from the logging guideline.
"Calling create device service, appId: {}, token: {}, platform: {} - start"

```
 logger.info("call={}, callType={}, action: createDevice, state: initiated, appId: {}, token: {}, platform: {}, environment: {}", "/push/device/create", "POST" ,  request.getAppId(), maskToken(request.getToken()), request.getPlatform(),request.getEnvironment())
...
 logger.info("call={}, callType={}, action: createDevice, state: succeeded", "/push/device/create", "POST");
```

We should add the environment.
The succeeded state log  should not print the data again (because of tracing)

Reply:
Ok, I will change the logging in this class.